src/docs/release/breaking-changes/1-22-deprecations.md

```diff
@@ -0,0 +1,768 @@
+ 
+#### Migration Guide
+ 
+Code before migration | Code after migration
+-- | --
+`InlineSpan.text` | `TextSpan.text`
+`InlineSpan.children` | `TextSpan.children`
+`InlineSpan.visitTextSpan` | `InlineSpan.visitChildren`
+`InlineSpan.recognizer` | `TextSpan.recognizer`
+`InlineSpan.describeSemantics` | `InlineSpan.computeSemanticsInformation`
+`PlaceholderSpan.visitTextSpan` | `PlaceHolderSpan.visitChildren`
+`TextSpan.visitTextSpan` |  `TextSpan.visitChildren`
+ 
+#### References
+ 
+API documentation:
+* [`InlineSpan`][]
+* [`TextSpan`][]
+* [`PlaceholderSpan`][]
+* [`WidgetSpan`][]
+ 
+Relevant issues:
+* [Text: support inline images][]
+
+Relevant PRs:
+* Development history:
+  * [#30069][]
+  * [#33946][]
+  * [#33794][]
+* Deprecated in [#34051][]
+* Removed in [#73747][]
+ 
+[`InlineSpan`]: {{site.api}}/flutter/painting/InlineSpan-class.html
+[`TextSpan`]: {{site.api}}/flutter/painting/TextSpan-class.html
+[`PlaceholderSpan`]: {{site.api}}/flutter/painting/PlaceholderSpan-class.html
+[`WidgetSpan`]: {{site.api}}/flutter/widgets/WidgetSpan-class.html
+[Text: support inline images]: {{site.github}}/flutter/flutter/issues/2022
+[#30069]: {{site.github}}/flutter/flutter/pull/30069
+[#33946]: {{site.github}}/flutter/flutter/pull/33946
+[#33794]: {{site.github}}/flutter/flutter/pull/33794
+[#34051]: {{site.github}}/flutter/flutter/pull/34051
+[#73747]: {{site.github}}/flutter/flutter/pull/73747
+
+---
+
+### `RenderView.scheduleInitialFrame`
+
+Supported by fix tool: No
+ 
+The `RenderView.scheduleInitialFrame` method was deprecated and removed in order to prevent
+splash screens from being taken down too early, resulting in a black screen. This would happen when
+`WidgetsFlutterBinding.ensureInitialized` was called. Instead, replace calls to this method with
+`RenderView.prepareInitialFrame`, followed by `RenderView.owner.requestVisualUpdate`.
+ 
+#### Migration Guide
+ 
+Code before migration:
+<!-- skip -->
+```dart
+scheduleInitialFrame();
+```
+ 
+Code after migration:
+<!-- skip -->
+```dart
+prepareInitialFrame();	
+owner.requestVisualUpdate();
+```
+ 
+#### References
+ 
+API documentation:
+* [`RenderView`][]
+* [`WidgetsFlutterBinding`][]
+ 
+Relevant issues:
+* [WidgetsFlutterBinding.ensureInitialized() takes down splash screen too early][]
+
+Relevant PRs:
+* Deprecated in [#39535][]
+* Removed in [#73748][]
+ 
+[`RenderView`]: {{site.api}}/flutter/rendering/RenderView-class.html
+[`TextSpan`]: {{site.api}}/flutter/widgets/WidgetsFlutterBinding-class.html
+[WidgetsFlutterBinding.ensureInitialized() takes down splash screen too early]: {{site.github}}/flutter/flutter/issues/39494
+[#39535]: {{site.github}}/flutter/flutter/pull/39535
+[#73748]: {{site.github}}/flutter/flutter/pull/73748
+
+---
+
+### `Layer.findAll`
+
+Supported by fix tool: No
+
+The `Layer.findAll` method was deprecated with the introduction of `Layer.findAnnotations` in order
+to unify the implementations of `find` and `findAll`. To migrate affected code, call on
```

Comment:
Nit: "call on" => "call"

If I'm reading this correctly.

Reply:
+1